app/buyers/views/buyers.py

```diff
@@ -115,7 +116,10 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     completed_sections = {}
     for section in sections:
         required, optional = count_unanswered_questions([section])
-        completed_sections[section.slug] = True if required == 0 else False
+        if section_has_at_least_one_required_question(section):
+            completed_sections[section.slug] = True if required == 0 else False
+        else:
+            completed_sections[section.slug] = True if optional == 0 else False
```

Comment:
Hmm... a thought.  Should be tick only if ALL optional are completed?
Or should it be `if optional < total_optional`?

At the moment it's a moot point, as the only section with all optional has just one question.  So let's leave it like this for now.